UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -981,6 +986,164 @@ public void ServerAGhost()
 		}
 	}
 
+	[Command]
+	public void CmdPlaceTile(string adminId, string adminToken, int categoryIndex, int tileIndex,
+		Vector3Int worldPosition, int matrixId, OrientationEnum orientation, Color? colour)
+	{
+		var admin = PlayerList.Instance.GetAdmin(adminId, adminToken);
```

Comment:
Curious why you decide to pass an admin ID and token and not find the user by connection as done elsewhere here? E.g. line `966` `if (AdminCommandsManager.IsAdmin(connectionToClient, out _))`

Reply:
I could move this to AdminCommandsManager